homeassistant/components/samsungtv/media_player.py

```diff
@@ -204,13 +204,17 @@ async def async_update(self) -> None:
             )
 
         if self._attr_state != STATE_ON:
+            if self._dmr_device:
+                await self._dmr_device.async_unsubscribe_services()
```

Comment:
Maybe we should shutdown the upnp_server completely here instead, and recreate it when the TV is online again.

```suggestion
                await self._async_shutdown_dmr()
```

Reply:
As long as it doesn't cause any problem without it, I don't think we should incur the overhead of shutdown/startups when the tv gets power cycled.  That could be expensive if you have a lot of them